doc/security/security-overview.rst

```diff
@@ -536,76 +536,188 @@ considered and documented.
-   prioritized and assigned to the owner of the affected module.
-   Additionally, the system security architect and the security
-   architect of the module are notified and shall take the
-   responsibility to mitigate the issue and review the solution or
-   counter-measure. In any case, the security issue shall be
-   documented centrally, including the affected modules, software
-   releases, and applicable workarounds for immediate mitigation. A
-   list of known security issues per public release of the Zephyr
-   shall be published and maintained by the Zephyr Security Subcommittee after a
-   risk assessment.
+Vulnerabilities to the Zephyr project may be reported via email to the
+vulnerabilities@zephyrproject.org mailing list.  These reports will be
+acknowledged and analyzed by the security response team within 1 week.
+Each vulnerability will be entered into the Zephyr Project security
+tracking JIRA_.  The original submitter will be granted permission to
+view the issues that they have reported.
+
+.. _JIRA: https://zephyrprojectsec.atlassian.net/
+
+Reporters may also submit reports by directly submitting them to the
+Zephyr Product security tracking JIRA.  Issues within this bug
+tracking system will transition through a number of states according
+to this diagram:
+
+.. figure:: media/zepsec-workflow.png
+
+- New: This state represents new reports that have been entered
+  directly by a reporter.  When entered by the response team in
+  response to an email, the issue shall be transitioned directly to
+  Triage.
+
+- Triage: This issue is awaiting Triage by the response team.  The
+  response team will analyze the issue, determine a responsible
+  entity, assign the JIRA ticket to that individual, and move the
+  issue to the Assigned state.  Part of triage will be to set the
+  issue's priority.
+
+- Assigned: The issue has been assigned, and is awaiting a fix by the
+  assignee.
+
+- Review: Once there is a Zephyr pull request for the issue, the PR
+  link will be added to a comment in the issue, and the issue moved to
+  the Review state.
+
+- Accepted: Indicates that this issue has been merged into the
+  appropriate branch within Zephyr.
+
+- Release: The PR has been included in a released version of Zephyr.
+
+- Public: The embargo period has ended.  The issue will be made
+  publically visible, the associated CVE updated, and the
+  vulnerabilities page in the docs updated to include the detailed
+  information.
+
+The issues created in this JIRA instance are kept private, due to the
+sensitive nature of security reports.  The issues are only visible to
+certain parties:
+
+- Members of the Zephyr Security Subcommittee
+
+- others, as proposed and ratified by the Zephyr Security
+  Subcommittee.  In the general case, this will include:
+
+  - The code owner responsible for the fix.
+
+  - The Zephyr release owners for the relevant releases affected by
+    this vulnerability.
+
+- the reporter
```

Comment:
wrong indentation.

Reply:
It's not, actually. There are three bullets at the top level, and the second one has two bullets under it.

Reply:
I'll move "the reporter" up so that it doesn't look like it might be a mistake.